src/content/pages/partnerships.mdx

```diff
@@ -0,0 +1,116 @@
+</Note>
+
+If you don't see your desired level of sponsorship listed, please contact us at `partner@astro.build`!
+
+We are always happy to work directly with larger sponsors ($2000+ per month) to create a custom sponsorship package. There are also limited options for being an exclusive "official" partner for a particular category (e.g. Official CMS Partner).
+
+## 📊 About Astro
+
+{/* The icons from the original are a nice touch! Can we use them here? e.g. trophy for #1 in retention */}
+
+<StatsFeature stats={[
+  { amount: "⬇ 240k", text: "monthly downloads" },
+  { amount: "🌐 79k", text: "X/Twitter Followers" },
+  { amount: "⭐ 48k", text: "GitHub Stars" },
+  { amount: "🏆 #1", text: "in developer retention" },
+  { amount: "❤ #1", text: "in developer interest" },
+  { amount: "📈 3x", text: "growth in usage" },
+]}/>
+
+## 🤝 Other ways to partner with Astro
+
+Astro maintains relationships with companies, content creators, and other open-source projects.
+
+Reach out to `partner@astro.build` to explore activities such as:
+
+- [Co-marketing](#-co-marketing)
+- Building or funding [feature development](#-development)
+- Providing a [technical case study](#-technical-case-studies)
+- Inviting us on [podcasts or live streams](#-podcasts-and-livestreams)
+- Creating [video content](#-videos-and-shorts)
+- Submitting [third-party guides](#-third-party-guides) to our docs site
+
+## 📣 Co-Marketing
+
+Co-marketing is partners helping each other by amplifying the others’ message, content, or resources to the benefit of both of communities. 
+
+If you create or build something with or featuring us, we’re happy to share it! This could take the form of a social media post or boost, a blog post announcing our joint-feature, or placement in our newsletter.
+
+For larger coordinated efforts, reach out to us to discuss a potential partnership with a financial arrangement.
+
+<Note color="purple">
+  ✍🏻 **To co-market:** Email partner@astro.build a brief summary of what you want us to amplify, where you want us to amplify it, and how it helps Astro users.
+</Note>
+
+## 🖥 Development
+
+Astro stays lightweight, environmentally friendly, and blazingly fast by being a small set of core features with minimal dependencies. That's why Astro's integration ecosystem is so powerful! Astro integrations extend basic functionality on an opt-in basis. 
+
+**Creating an Astro integration for your product or service is a great way to make it easy for people to use you in their Astro project, and grows the entire ecosystem!**
+
+Don't have the time or people to build your own Astro integration for your company or product? You can partner with us to **directly fund development on an integration**. This allows Astro to prioritize the functionality you need, and create a plug-and-play way for Astro projects to use your service.
```

Comment:
```suggestion
Don't have the time or people to build your own Astro integration for your company or product? You can partner with us to **directly fund development of an integration**. This allows Astro to prioritize the functionality you need, and create a plug-and-play way for Astro projects to use your service.
```
Not sure

Reply:
I think either could work, but I like yours better!